inference_perf/main.py

```diff
@@ -63,25 +64,35 @@ def main_cli() -> None:
         raise Exception("load config missing")
 
     # Define Metrics Client
-    if config.metrics:
-        metricsclient = MockMetricsClient(uri=config.metrics.url)
-    else:
-        raise Exception("metrics config missing")
+    metrics_client: MetricsClient | None = None
+    if config.metrics_client:
+        if config.metrics_client.type == MetricsClientType.PROMETHEUS:
+            metrics_client = PrometheusMetricsClient(metrics_client_config=config.metrics_client)
```

Comment:
This will simplify the client code
```suggestion
            metrics_client = PrometheusMetricsClient(metrics_client_config=config.metrics_client.prometheus)
```

Reply:
done